wdl2cwl/tests/cwl_files/BuildCembaReferences.cwl

```diff
@@ -0,0 +1,395 @@
+class: Workflow
+id: BuildCembaReferences
+inputs:
+  - id: Convert.fasta_input
+    type: File
+  - id: Convert.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: IndexForward.fasta_input
+    type: File
+  - id: IndexForward.index_prefix
+    type: string
+  - id: IndexForward.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: IndexReverse.fasta_input
+    type: File
+  - id: IndexReverse.index_prefix
+    type: string
+  - id: IndexReverse.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: CreateReferenceDictionary.reference_fasta
+    type: File
+  - id: CreateReferenceDictionary.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: CreateReferenceFastaIndex.reference_fasta
+    type: File
+  - id: CreateReferenceFastaIndex.monitoring_script
+    type:
+      - File
+      - 'null'
+outputs:
+  - id: Convert.fwd_converted_reference_fasta_output
+    outputSource: Convert/fwd_converted_reference_fasta_output
+    type: File
+  - id: Convert.rev_converted_reference_fasta_output
+    outputSource: Convert/rev_converted_reference_fasta_output
+    type: File
+  - id: Convert.monitoring_log
+    outputSource: Convert/monitoring_log
+    type: File
+  - id: IndexForward.bowtie2_index_files
+    outputSource: IndexForward/bowtie2_index_files
+    type:
+        items: File
+        type: array
+  - id: IndexForward.monitoring_log
+    outputSource: IndexForward/monitoring_log
+    type: File
+  - id: IndexReverse.bowtie2_index_files
+    outputSource: IndexReverse/bowtie2_index_files
+    type:
+        items: File
+        type: array
+  - id: IndexReverse.monitoring_log
+    outputSource: IndexReverse/monitoring_log
+    type: File
+  - id: CreateReferenceDictionary.ref_dict_output
+    outputSource: CreateReferenceDictionary/ref_dict_output
+    type: File
+  - id: CreateReferenceDictionary.monitoring_log
+    outputSource: CreateReferenceDictionary/monitoring_log
+    type: File
+  - id: CreateReferenceFastaIndex.ref_index_output
+    outputSource: CreateReferenceFastaIndex/ref_index_output
+    type: File
+  - id: CreateReferenceFastaIndex.monitoring_log
+    outputSource: CreateReferenceFastaIndex/monitoring_log
+    type: File
```

Comment:
Likewise the original WDL file has 6 defined outputs
```
ipdb> print(len(obj.effective_outputs))
6
```
```suggestion
outputs:
  - id: fwd_converted_reference_fasta_output
    outputSource: Convert/fwd_converted_reference_fasta_output
    type: File
  - id: rev_converted_reference_fasta_output
    outputSource: Convert/rev_converted_reference_fasta_output
    type: File
  - id: fwd_bowtie2_index_files
    outputSource: IndexForward/bowtie2_index_files
    type:
        items: File
        type: array
  - id: rev_bowtie2_index_files
    outputSource: IndexReverse/bowtie2_index_files
    type:
        items: File
        type: array
  - id: reference_fasta_dict
    outputSource: CreateReferenceDictionary/ref_dict_output
    type: File
  - id: reference_fasta_index
    outputSource: CreateReferenceFastaIndex/ref_index_output
    type: File
```

Reply:
I made a small edit above to match the output of `miniwdl run` by prepending each name with `BuildCembaReferences.`